release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2361,7 +2365,7 @@ In the following tables, features are marked with the following statuses:
 |AWS Load Balancer Operator
 |Not Available
 |Technology Preview
-|General Availability
+|Technology Preview
```

Comment:
@skrthomas can you explain this change? I looked at the preview but couldn't figure it out myself :sweat_smile:

Reply:
Yeah, I noticed that someone else, not me, put Network Observability in this table as technology preview in 4.12. I'm not sure _who_ added it but its wrong, obviously, we GA'd in 4.12. So I'm updating this table. Then the Operator also appears in the 4.13 version of this table, and its still General Availability, so I persisted the "GA" in the 4.13 column as well.

Reply:
is this our change? I thought this pertains to `AWS Load Balancer Operator`

Reply:
This is not our change. The release notes file is so murky to work in because all the writers in OCP are updating this file. There's always rebasing needed and changes like this can sneak in. I'm going to remove this.